src/dirigera/devices/scene.py

```diff
@@ -64,6 +64,25 @@ class Trigger(BaseIkeaModel):
     type: str
```

Comment:
```python
id: Optional[str] = None
```
Should be optional because of validation error: 
```bash
Traceback (most recent call last):
  File "/Users/slajob/git/dirigera_local/shortcut-somrig.py", line 31, in <module>
    Trigger(type="app", disabled=False),
  File "/Users/slajob/git/dirigera_local/venv/lib/python3.9/site-packages/pydantic/main.py", line 164, in __init__
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 1 validation error for Trigger
id
  Field required [type=missing, input_value={'type': 'app', 'disabled': False}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.4/v/missing
```